docs/content.zh/docs/ops/metrics.md

```diff
@@ -28,26 +28,26 @@ under the License.
 
 # 指标
 
-Flink exposes a metric system that allows gathering and exposing metrics to external systems.
+Flink 公开了一个指标系统，允许收集和公开指标给外部系统。
 
-## Registering metrics
+## 注册指标
 
-You can access the metric system from any user function that extends [RichFunction]({{< ref "docs/dev/datastream/user_defined_functions" >}}#rich-functions) by calling `getRuntimeContext().getMetricGroup()`.
-This method returns a `MetricGroup` object on which you can create and register new metrics.
+你可以通过调用从 [RichFunction]({{< ref "docs/dev/datastream/user_defined_functions" >}}#rich-functions) 
+扩展的任何用户函数的 `getRuntimeContext().getMetricGroup()` 方法访问指标系统。
+此方法返回一个 `MetricGroup` 对象，你可以在该对象上创建和注册新指标。
 
-### Metric types
+## 指标类型
```

Comment:
reduce a '#' ?
Could you give a explain for the chage ? 
Glad to see the reason.